### PR TITLE
first commit for cobigen setup

### PR DESCRIPTION
**Pull request by dixyushi**

### CODE REVIEW

**Comment by GuentherJulian** [comment]
The parser expects at least one step. Therefore the pipeline fails at the moment

File: cobigen-setup/index.asciidoc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+[step]
+--
+restoreDevonfwIde(["java","mvn"])
+--
+
+For this tutorial an installation of the devonfw IDE is required, which was already set up for you.
+To begin you need to install CobiGen and create a Java Project.
+[step]
+== Install CobiGen and create Java Project
```
Comment (SchettlerKoehler) [inline cobigen-setup/index.asciidoc:18]: This is a syntax error. See https://github.com/devonfw-tutorials/tutorials/wiki/Development

File: cobigen-setup/index.asciidoc [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+--
+installCobiGen()
+--
+
+====
```
Comment (SchettlerKoehler) [inline cobigen-setup/index.asciidoc:23]: This is a syntax error. See https://github.com/devonfw-tutorials/tutorials/wiki/Development

**Review by SchettlerKoehler — CHANGES_REQUESTED** [review]

**Comment by dixyushi** [comment]
I have created new drafted pull request. This branch is deleted because of issue which were created earlier. Hence working on one branch and closing this pull request

**Pull request closed without merging**